### ISSUE TITLE
F-Droid version out of date

### ISSUE THREAD

**Issue by tuch96** [Digitaler-Impfnachweis/covpass-android#141]
What is happening with the version on f-droid? This has not been updated for months.

**Comment by Ein-Tim** [comment]
The F-Droid apps repo is located on GitLab: https://gitlab.com/covpass-android-foss/covpass-android

**Comment by jleufgen** [comment]
@tuch96 could we close this ticket?

**Comment by tuch96** [comment]
@jleufgen Of course! @Ein-Tim Unfortunately there is no feedback from the team at Gitlab on how the f-droid version is progressing.

**Comment by wkornewald** [comment]
To provide some more context: The F-Droid app is not officially maintained by us. Some people started a fork and apparently then gave up maintaining it. We currently only provide official Play Store apps and I don't know if we'll ever provide F-Droid apps (though I still have hope...).

**Comment by tuch96** [comment]
@wkornewald thanks for your reply!! It would be a wonderful sign for an official f-droid version. A sign for alternative app stores.

**Comment by wkornewald** [comment]
@tuch96 Like I said, I don't know if it'll ever happen.

**Comment by github-tomster** [comment]
@tuch96 
sit down, take a deep breath an read on here :
https://github.com/Digitaler-Impfnachweis/covpass-android/issues/28
https://github.com/Digitaler-Impfnachweis/covpass-android/issues/120

feel free to 😭 
this is 🇩🇪 !
oh and we have to keep the tone fine and down. 💢 🗯️ 

the upper-level federal agency driven by public taxes <- am i wrong on that ❓ 
is not able/unwilling to provide a **free-accessible apk** to the public 🤦🏻‍♂️ 
i am close to the point to think they are violating german law with this attitude 👮🏻‍♂️ 

however, check here:
https://f-droid.org/en/packages/ch.admin.bag.covidcertificate.wallet/
https://github.com/admin-ch/CovidCertificate-App-Android/releases
https://github.com/admin-ch/CovidCertificate-App-Android
maybe this helps you to see: it is possible an the only acceptable way 🥇

**Comment by andycandy-de** [comment]
@jleufgen Can' t you answer why there is no official open souce to download the APK? #120
The official app of swiss is also available in the F-Droid Store https://f-droid.org/de/packages/ch.admin.bag.covidcertificate.wallet/
This is very exemplary.

Why do you force people to use commercial appstores? I would like to understand the decision.

**Comment by jugendhacker** [comment]
> To provide some more context: The F-Droid app is not officially maintained by us. Some people started a fork and apparently then gave up maintaining it. We currently only provide official Play Store apps and I don't know if we'll ever provide F-Droid apps (though I still have hope...).

Just to be more precise here: we did not give up maintaining it, but have a bus factor = 1 problem currently. We are working on getting this resolved and hopefully get it updated in the near future. Meanwhile this should definitely be closed here as IBM is not responsible as @wkornewald already said.

**Comment by tuch96** [comment]
@jugendhacker thanks for your response to this topic and an overview of the current status of the project.

**Comment by alexcimander** [comment]
Hey everyone,
first of all thank you for your honest feedback and the discussions. As @wkornewald already mentioned the F-Droid version of the app is not maintained by us and therefore we cannot guarantee that the version is up-2-date. 

Our efforts to publish the apps via F-Droid were unfortunately initially not prioritized in comparison to other topics. Saying that we're working hard to define the best priorities.

Having an unmaintained version out there is a potential risk since we are working nonstop to keep the app up with the pace of the pandemic and regulations. With that in mind we will definitely try to raise awareness and for sure we are willing to bring it back to discussion. We will provide you an update as soon as we know more.

Once again: Thank you for participating.

**Comment by alexcimander** [comment]
Thank you @jugendhacker for the update as well!

**Comment by andycandy-de** [comment]
> Our efforts to publish the apps via F-Droid were unfortunately initially not prioritized in comparison to other topics. Saying that we're working hard to define the best priorities.

It's really sad how ignorant this decision is against all people who want to download the app from an open source. How high did you estimate the efford to puplish the APK to other platforms (github or F-Droid)?

> Having an unmaintained version out there is a potential risk since we are working nonstop to keep the app up with the pace of the pandemic and regulations.

That's totally true! Please rethink priosing and don't make people dependent on global companies. This affects your privacy promises.

**Comment by Excpt0r** [comment]
@alexcimander 
+1 for an officially supported f-droid version
I was not able to scan a QR code with an outdated version from f-droid, that also had a complete different versioning scheme.
F-Droid is the first place where I search for apps.

But just for the record, I saw there is a new f-droid release since yesterday, that also seems to align the versioning scheme.
(Didn't test it)

**Comment by Ein-Tim** [comment]
The[F-Dorid Repo](https://f-droid.org/de/packages/de.rki.covpass.app/) shows an update today:

Version 1.13.2 (711) **Hinzugefügt am 2022-01-08**

It seems like Google Play Store and F-Dorid are in sync again.

**Comment by Ein-Tim** [comment]
Can this issue be closed now @tuch96?

**Comment by tuch96** [comment]
@Ein-Tim The ticket can be closed. Thanks.

**Comment by Ein-Tim** [comment]
@tuch96 

You, as the author, can close this issue (-:

**Comment by andycandy-de** [comment]
Is the main issue really solved? Did you decide to publish the app to f-droid and/or other open public sources?
If not you still force the people to use commercial app stores to install the official app and the F-Droid Fork will be outdated any time soon ...

**Comment by Ein-Tim** [comment]
@andycandy-de 

This issue is not about making F-Droid versions of the app, but that the versions of the app on F-Droid were out of date. 
Thus, I vote for closing this issue. 
Other topics should be discussed in a separate issue.
